src/test/datascience/notebook/helper.ts

```diff
@@ -161,7 +158,10 @@ export async function createTemporaryNotebook(templateFile: string, disposables:
 }
 
 export async function canRunNotebookTests() {
-    if (!isInsiders() || !process.env.VSC_JUPYTER_RUN_NB_TEST) {
+    if (
+        //isInsiders() ||
+        !process.env.VSC_JUPYTER_RUN_NB_TEST
```

Comment:
All tests can run in stable & insiders, this prevented tests from running even in stable in release branch.